Unic.Bob.Keith.nuspec

```diff
@@ -12,6 +12,8 @@
     </metadata>
     <files>
       <file src="src\**" />
-      <file src="packages\**" target="tools\" />
+      <file src="packages\LibGit2Sharp\App_Readme\**" target="tools\LibGit2Sharp\App_Readme\" />
+      <file src="packages\LibGit2Sharp\lib\**" target="tools\LibGit2Sharp\lib\" />
+      <file src="packages\Microsoft.AspNet.Razor\lib\**" target="tools\Microsoft.AspNet.Razor\lib\" />
```

Comment:
Maybe `exclude="**\package\services\metadata\core-properties\**` on ` <file src="packages\**"` would be the better pattern

Reply:
As discussed will keep the version of libgit2sharp and also keep the less-inclusive pattern for the packaging (at least until we re-implement the Bob the Book generation part) :)